model/app/webapp.go

```diff
@@ -642,9 +623,18 @@ func ListWebappsWithPagination(db prefixer.Prefixer, limit int, startKey string)
 	// - There are no more docs in couchDB
 	// - There are no docs at all
 	// We can load extra apps and append them safely to the list
-	for slug := range appsdir {
-		if man, err := loadManifestFromDir(slug); err == nil {
-			docs = append(docs, man)
+	if RegistryFS != nil {
+		for _, slug := range RegistryFS.List() {
+			appMan, err := RegistryFS.GetWebAppManifest(slug)
+			if errors.Is(err, ErrNotFound) {
+				continue
```

Comment:
This case should never happen since we're looping over the list of apps in the registry.
So I think we should remove this block for 2 reasons:
- it adds seemingly unnecessary logic
- if it ever happens, we have a problem so we should return the error